pkg/certificate/certificate.go

```diff
@@ -19,7 +20,7 @@ func NewProvider(pt ProviderType) (certInterface.Provider, error) {
 	case Auto:
 		return nil, nil // change me later
 	case CertManager:
-		return nil, nil // change me later
+		return &certManager.CertManagerProvider{}, nil // change me later
```

Comment:
Is there anything that can't be finalized yet?
```suggestion
		return certManager.New(...), nil
```